docs/useAuthState.md

```diff
@@ -5,12 +5,13 @@ title: "useAuthState"
 
 # `useAuthState`
 
-To avoid rendering a component, and to force waiting for the `authProvider` response, use `useAuthState()` instead of `useAuthenticated()`. It calls `authProvider.checkAuth()` on mount and returns an object with 2 properties:
+To avoid rendering a component, and to force waiting for the `authProvider` response, use `useAuthState()` instead of `useAuthenticated()`. It calls `authProvider.checkAuth()` on mount and returns a state object:
```

Comment:
```suggestion
To avoid rendering a component, and to force waiting for the `authProvider` response, use `useAuthState()` instead of [`useAuthenticated()`](./useAuthenticated.md). It calls [`authProvider.checkAuth()`](./AuthProviderWriting.md#checkauth)  on mount and returns a state object:
```